wandb/integration/keras/keras.py

```diff
@@ -408,6 +408,15 @@ def __init__(
         wandb.save("model-best.h5")
         self.filepath = os.path.join(wandb.run.dir, "model-best.h5")
         self.save_model = save_model
+        if save_model:
+            wandb.termwarn(
```

Comment:
If you're think we should plan on deprecating this, please deprecate this following this guide: https://github.com/wandb/client/blob/master/CONTRIBUTING.md#deprecating-features